test_chime.py

```diff
@@ -1,12 +1,30 @@
+import importlib
+import pathlib
+import platform
 import subprocess
+import tempfile
+import textwrap
 import time
 import typing
 
+import _pytest.monkeypatch
 import pytest
 
 import chime
 
 
+@pytest.fixture(scope='function', autouse=True)
+def reload_chime():
+    importlib.reload(chime)
+
+
```

Comment:
So there was some fragility around testing the theme.

First of all, if one test modifies `THEME`—like `test__get_config_path` currently does—this could end up altering the state of some tests. So if a test ends on chaining the theme to `"zelda"`, and the next test ran `chime.theme()`, it would return `"zelda"`.

`reload_chime` just reloads chime before every test to reset the state in an attempt to fix this.